units/patch_intent.go

```diff
@@ -227,6 +227,10 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 		return errors.New("patching is disabled for project")
 	}
 
+	if patchDoc.IsBackport() && !pref.CommitQueue.Enabled {
```

Comment:
Are backports also commit queue patches? Then IsCommitQueuePatch would also work

Reply:
I didn't intent backports to be commit queue patches.
The idea is to 1) create a regular patch on the new project and if it succeeds 2) to create and enqueue a commit queue patch.